sm3/sm3_disasm.h

```diff
@@ -27,11 +27,17 @@ class Disassembler {
   /** Disassemble instruction into a string. */
   std::string disassembleOp(const Instruction& op);
 
+  void setConstantTable(ConstantTable&& ctab) {
```

Comment:
Can't we parse constant names when encountering a comment token, or are those not typically early in the binary?

Either way, not a huge fan of passing this in externally.

Reply:
In that case we should just set up ctab metadata as the binary is being traversed.